source/developer/slash-commands.rst

```diff
@@ -53,6 +53,12 @@ Each Mattermost installation comes with some built-in slash commands that are re
     "/settings", "Open the Account Settings dialog", "/settings"
     "/shortcuts", "Display a list of keyboard shortcuts", "/shortcuts"
     "/shrug *{message}*", "Add ``¯\_(ツ)_/¯`` to your message", "/shrug oh well"
+    "/status *{emoji_name}* *{descriptive status_message}* *", Set a custom status that includes an optional emoji and a descriptive status message, "/status sick Feeling unwell and taking time off to recover"
```

Comment:
```suggestion
    "/status *{emoji_name}* *{descriptive status_message}*", "Set a custom status that includes an optional emoji and a descriptive status message", "/status sick Feeling unwell and taking time off to recover"
```

Reply:
I'm not sure if this resolves the issue (which was that the whole addition was italicized) - I think it'll only display once the change is committed.